docs/design/actions/move.md

```diff
@@ -0,0 +1,29 @@
+# movePod #
+[This project](https://github.com/songbinliu/movePod) demonstrates a method that can move pods, which either are created by ReplicationController, or by ReplicaSet(which may be created by Deployment).
```

Comment:
change to

[This project](https://github.com/songbinliu/movePod) demonstrates a method that moves pods, which are either created by ReplicationController, or by ReplicaSet(which may be created by Deployment).